spec/exasol/vscl/queryrenderer/ExpressionAppender_spec.lua

```diff
@@ -4,16 +4,28 @@ local Query = require("exasol.vscl.Query")
 local literal = require("exasol.vscl.queryrenderer.literal_constructors")
 local reference = require("exasol.vscl.queryrenderer.reference_constructors")
 local ExpressionAppender = require("exasol.vscl.queryrenderer.ExpressionAppender")
+local AbstractQueryAppender = require("exasol.vscl.queryrenderer.AbstractQueryAppender")
 
-local function assert_expression_yields(expression, expected)
-    assert.append_yields(ExpressionAppender, expected, expression)
+---@param expression any
+---@param expected string
+---@param appender_config AppenderConfig?
+local function assert_expression_yields(expression, expected, appender_config)
```

Comment:
The method signature is not ideal, since the name does not indicate that an appender config is the last parameter. That being said, it's clear enough.

Reply:
I know that is clunky, but a solution would be:
```java
assert_expression_yields_with_given_config(expression, expected, appender_config)
```